src/PrestaShopBundle/Resources/views/Admin/WebProfiler/config.html.twig

```diff
@@ -100,8 +100,8 @@
             <div class="sf-toolbar-info-piece">
                 <b>Resources</b>
                 <span>
-                    {% set appVersion = collector.applicationversion|split('.')|join()[:2] %}
-                    <a href="http://doc.prestashop.com/display/PS{{ appVersion }}" rel="help">
+                    {% set appVersion = collector.applicationversion|join()[:3] %}
+                    <a href="https://devdocs.prestashop.com/{{ appVersion }}/" rel="help">
```

Comment:
Computed link is `https://devdocs.prestashop.com/1.7/` for PS 1.7 versions. For next major, it will be `https://devdocs.prestashop.com/8.0/`